src/fidocadj/dialogs/DialogOptions.java

```diff
@@ -557,90 +563,287 @@ private JPanel createPCBsizePanel()
         pcbpadheight.setText("" + pcbpadheight_i);
-        constraints = DialogUtil.createConst(1, 2, 1, 1, 100, 100,
+        constraints = DialogUtil.createConst(1, 2, 1, 1, 0.5, 0.01,
                 GridBagConstraints.WEST, GridBagConstraints.NONE,
-                new Insets(6, 6, 6, 6));
+                new Insets(8, 6, 6, 6));
         pcbSizePanel.add(pcbpadheight, constraints);  // Add pcbline height tf
 
         JLabel pcbpadintwlbl = new JLabel(Globals.messages.getString(
                 "pcbpad_intw"));
-        constraints = DialogUtil.createConst(0, 3, 1, 1, 100, 100,
+        constraints = DialogUtil.createConst(0, 3, 1, 1, 0.5, 0.01,
                 GridBagConstraints.EAST, GridBagConstraints.NONE,
-                new Insets(6, 6, 6, 6));
+                new Insets(8, 6, 6, 6));
         pcbSizePanel.add(pcbpadintwlbl, constraints);// Add pcbpad int w label
 
         pcbpadintw = new JTextField(10);
         pcbpadintw.setText("" + pcbpadintw_i);
-        constraints = DialogUtil.createConst(1, 3, 1, 1, 100, 100,
+        constraints = DialogUtil.createConst(1, 3, 1, 1, 0.5, 0.01,
                 GridBagConstraints.WEST, GridBagConstraints.NONE,
-                new Insets(6, 6, 6, 6));
+                new Insets(8, 6, 6, 6));
         pcbSizePanel.add(pcbpadintw, constraints);      // Add pcbline width tf
 
+        constraints = DialogUtil.createConst(0, 4, 2, 1, 0.0, 1.0, 
+            GridBagConstraints.CENTER, GridBagConstraints.BOTH, 
+            new Insets(0, 0, 0, 0));
+        pcbSizePanel.add(new JLabel(), constraints);
+        
         return pcbSizePanel;
     }
 
-    /** Creates the panel dedicated to the extensions introduced by FidoCadJ on
-     * the original FidoCAD file format.
+    /** Creates the panel dedicated to the theme settings
      *
-     * @return the panel concerning extensions to the very old FidoCAD format.
+     * @return the panel containing the theme settings.
      */
-    private JPanel createExtensionsPanel()
+    private JPanel createThemePanel()
     {
-        /** ********************************************************************
-         * FidoCadJ extensions
-         **********************************************************************/
-        JPanel extensionsPanel = new JPanel();
+        JPanel themePanel = new JPanel();
+        themePanel.setLayout(new GridBagLayout());
+        themePanel.setOpaque(false);
 
         GridBagConstraints constraints = new GridBagConstraints();
-        extensionsPanel.setLayout(new GridBagLayout());
-        extensionsPanel.setOpaque(false);
 
-        extStrict_CB = new JCheckBox(Globals.messages.getString(
-                "strict_FC_comp"));
-        extStrict_CB.setSelected(extStrict);
-        extStrict_CB.setOpaque(false);
+        // Groupbox "Theme Management"
+        JPanel themeManagementPanel = new JPanel();
+        themeManagementPanel.setBorder(
+                BorderFactory.createTitledBorder("Gestione tema"));
```

Comment:
Is that normal that resources in Italian appear here?

Reply:
It was fixed in the subsequent commits... **(Outdated)**

Reply:
Ok, sorry, I was looking at the commits and reading the code for each modification. Sometimes the review system here is not crystal clear to follow.

Reply:
No problem, you have quite a bit to review today! :-)